src/BrowserFetcher.ts

```diff
@@ -219,6 +229,10 @@ export class BrowserFetcher {
     if (await existsAsync(outputPath)) return this.revisionInfo(revision);
     if (!(await existsAsync(this._downloadsFolder)))
       await mkdirAsync(this._downloadsFolder);
+    if (os.arch() === 'arm64') {
+      await handleArm64();
```

Comment:
No need to `await` since it doesn't return a promise:

```suggestion
      handleArm64();
```